build.xml

```
@@ -59,7 +59,7 @@
               passthru="true"/>
 
         <!-- Build frontend assets -->
-        <exec dir="${build.dir}" command="yarn install &amp;&amp; yarn build" passthru="true" checkreturn="true"/>
+        <exec dir="${build.dir}" command="yarn install --ignore-scripts &amp;&amp; yarn build --ignore-scripts" passthru="true" checkreturn="true"/>
```

Tohle nám stačí jen u instalace, ne?

```suggestion
        <exec dir="${build.dir}" command="yarn install --ignore-scripts &amp;&amp; yarn build" passthru="true" checkreturn="true"/>
```